lib/workers/repository/update/pr/index.ts

```diff
@@ -82,16 +107,30 @@ export async function ensurePr(
   );
 
   const config: BranchConfig = { ...prConfig };
-
+  const prFingerprint = fingerprint(config);
   logger.trace({ config }, 'ensurePr');
   // If there is a group, it will use the config of the first upgrade in the array
   const { branchName, ignoreTests, prTitle = '', upgrades } = config;
   const dependencyDashboardCheck =
     config.dependencyDashboardChecks?.[config.branchName];
-  // Check if existing PR exists
+  // Check if PR already exists
   const existingPr = await platform.getBranchPr(branchName);
   if (existingPr) {
     logger.debug('Found existing PR');
+    const prCache = getPrCache(branchName);
+    if (prCache) {
+      logger.debug({ prCache }, 'Found existing PR cache');
```

Comment:
```suggestion
      logger.trace({ prCache }, 'Found existing PR cache');
```

Don't think we want this level of detail printed forever?